modules/custom/cu_core/cu_core.module

```diff
@@ -4,6 +4,15 @@
  * Contains hooks for cu_core module.
  */
 
+/**
+ * Implements hook_menu_alter().
+ */
+function cu_core_menu_alter(&$items) {
+  // This is a place to remove visibility for enpoints
+  // unnecessarily set as public.
+  $items['filter/tips']['access callback'] = FALSE;
```

Comment:
I think we still want editors to be able to see this page, maybe wrap in a check for `authenticated`?